atk.yml

```diff
@@ -49,11 +51,19 @@ services:
       args:
         IMAGE_BASE: "ubuntu"
         IMAGE_TAG: "22.04"
-        APT_DEPENDENCIES: "bash zsh vim git git-lfs python3-pip python3-tk python3-opencv ros-${ROS_DISTRO}-usb-cam ros-${ROS_DISTRO}-tf-transformations ros-${ROS_DISTRO}-rqt-graph python3-pil.imagetk"
-        PIP_REQUIREMENTS: "pandas matplotlib numpy>=1.19 opencv-python tornado black Pillow torch torchvision pyserial nvidia-pyindex nvidia-tensorrt transforms3d scipy"
         USER_GROUPS: "dialout video"
         USER_SHELL_ADD_ONS: ". /home/${COMPOSE_PROJECT_NAME}/${COMPOSE_PROJECT_NAME}/workspace/install/setup.bash"
+        PIP_REQUIREMENTS: "tensorrt"
+        ROSDEP_METAPACKAGE: "src/common/meta/art_dev_meta/"
     working_dir: "/home/${COMPOSE_PROJECT_NAME}/${COMPOSE_PROJECT_NAME}/workspace"
+  art-5:
```

Comment:
assuming this is a precursor to #60? What does this do here? I don't see any other references to `art_art5_meta` added in this PR, I may just be blind though...

Reply:
more asking out of curiosity, this doesn't have to change.

Reply:
Removed, it's just going to be in #60